### DISCUSSION TITLE
Managing AI on the supercarrier

### DISCUSSION THREAD

**Discussion by JF-JEFX** [Khopa/dcs_liberation#1951] [Q&A] [answered]
I dont really know where to put this discussion (since most of the problems we encounter on the carrier are DCS limitation and AI's stupidity... but still, here are a few thoughts to consider.  

I have been playing Liberation mostly from an F/A-18C and Supercarrier perspective and it is quite difficult to manage from the carrier operations standpoint.  I like to start campaigns with AI default start to COLD, since that gives much more variety in OCA strike missions and more realism around myself... But... on the carrier, it is very difficult for several reasons..
1) if one has some F-14 squadrons, the Tomcats are quite big and, when taxiing, get in the way and block each other...
2) our own aircraft always spawns in the area in the middle of the place, blocking several other aircraft from taxiing, especially the ones from the back of the carrier (often bumping into us and making the mission fail...) 
3) the AWACS (E-2D) gets stuck for a considerable amount of time behind other spawning AC

I have played Liberation with default start IN FLIGHT to solve all this and it works but removes a lot of the fun of carrier operations when you are alone on the deck...

More recently, I tried something that kind of works... I have to add that I rarely use a wingman since their behavior is so bad... that explains how I can do what I will describe here: (as a solo aircraft)

When I start a new (naval) campaign, I remove the F-14 squadrons and replace them with en equivalent number of F/A-18C squadrons.  I assign the default start time to COLD, except for my own flight which I edit to WARM.  The minute I spawn (since in a warm start the INS is already aligned) I move out of the way: I tried several spots, but the best one (without anyone spawning on top of me) is the very stern end of the boat, behind the wires, since nobody will try and trap at the start of a mission.  When I move there, suddenly all other AI flights are able to spawn, taxi and launch properly (especially since they are all F-18S and no Tomcats).  In this morning's experiment, I managed to have all 40 aircraft that were slotted in my ATO for the first 18 minutes of my Liberation turn to launch in about 27 minutes with no problem, it is a bit long, but it is the first time I manage to get a boatful of Aircraft to spawn and launch without taxi glitches...

That brings me to my suggestion for a feature request... Would it be possible in some future iteration of Liberation to have control over the default start type with more options?  perhaps the first one that pops to mind is per coalition (BLUE can start IN FLIGHT to minimize taxing problems and RED could start COLD to maximize the fun of OCA strike missions? or more specialized, all could spawn COLD but only carrier based aircraft could spawn RUNWAY or IN FLIGHT in order to fix the taxiing mess ??

Thanks, Liberation is so Awesome (just finished a campaign yesterday, Operation Blackball, in 18 turns, and it was a real blast!!!!)

**Comment by Starfire13** [comment]
Since this is a feature request, it would be better to file this as a feature request. That said, I have found that as long as you space all your flights out (3 minutes is the absolute minimum), they don't get stuck on the carrier. Essentially, you want the previous flight to have already taken off by the time the next flight starts taxiing to the cats.

**Comment by JF-JEFX** [comment]
Thanks Starfire for your answer, and I see what you mean.  Perhaps, my experimentation lead me to think here out loud... but, yes, it would be perhaps a feature request, but I dont want to double post, feel free to move it around if you like, thanks.

**Reply by Starfire13** [reply]
I think you can click the "Create Issue from Discussion" button to convert it.

**Comment by JF-JEFX** [comment]
Sorry Starfire, it doesn't work... I tried the 'Create issue from Discussion' button and it doesn't convert anything, only starts a new issue... But I dont understand, you are talking about a 'feature request' and I dont find this menu in GitHub?  perhaps I am not very familiar with GitHub but I see only 'issues' and 'discussions'  where is the 'feature request' page?

**Reply by Starfire13** [reply]
Click on Issues tab, click New Issues buttoon, click Feature Request button. But you can simply click on the Create New Issue from Discussion button and then click the Feature Request button. And then fill in the template from there.

**Reply by pcoud** [reply]
Hi JF-JEFX, there is a simple way to solve your point 2. The "area in the middle of the place" as you call it, is actually called "the six pack". In DCS you can only spawn at the six pack at the beginning of the mission, ie if you spawn at t0 + 1sec you can be sure you won't be at the six pack. So: just start your mission as an observer, then esc ---> choose slot, jump in your plane and enjoy your cold start with a lot of activity around, and, if you follow starfire advice (space AI flights a little bit), virtually no more traffic jams.

**Comment by JF-JEFX** [comment, marked as answer]
Thanks pcoud  that sounds like a great bit of advice, never thought of doing this!  Will try it, and yes, Starfire's advice about spacing the flights with a few minutes between each is very good too! thanks!

**Reply by JF-JEFX** [reply]
really works well, thanks so much!!
